pillar.example

```diff
@@ -1,4 +1,14 @@
 letsencrypt:
+  use_package: true                              # Install using packages instead of git
+  pkg: python-certbot-apache                     # Package name to use. Check https://certbot.eff.org/all-instructions
+                                                 # to find the correct name for the variant you want to use.
+  add_individual_crons: false                    # certbot's package uses a systemd's timer unit to try to renew the
```

Comment:
Do we want one more setting, or could we assume that this behaviour is expected when using _use_package_ set to _true_ ? Moreover, when _add_individual_crons_ is set to _false_, we could also remove the cron entries to make the transition easier?

Reply:
When using the package, certbot installs a systemd timer unit, so it will try to refresh all certs. I think that we can safely assume that, when using package, the individual crons will be removed (if still declared in the pillar)? OTOH, when installing via git, the declared crons will be installed. And that's it, we can get rid of *add_individual_crons*.

I added it just in case the user of the formula wanted the crons there. But with the removal of the scripts above, we can safely assume that the crons are unneeded.